src/neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -273,30 +272,22 @@ def get_metadata_container(
         container_id: Union[UniqueId, QualifiedName],
         expected_container_type: typing.Optional[ContainerType],
     ) -> ApiExperiment:
-        try:
-            experiment = (
-                self.leaderboard_client.api.getExperiment(
-                    experimentId=container_id,
-                    **DEFAULT_REQUEST_KWARGS,
-                )
-                .response()
-                .result
+        experiment = (
```

Comment:
If I understand correctly, this could now throw a `RuntimeError` - don't we want to catch it and throw the domain exception instead?

Reply:
As far as I can understand, we've removed management functionality so those errors will never occur so we don't have to worry about them for now.